deployment/ansible/roles/ckan-odp-configuration/tasks/letsencrypt.yml

```diff
@@ -30,7 +30,7 @@
     name: "Renew Let's Encrypt Certificate"
     minute: 0
     hour: 23
-    day: 25
+    day: "1,15"
```

Comment:
Current certificate expires 5/9/2021. I didn't make any effort to try to match this up with an expiry date since we're going to be running this every two weeks anyway.